docs/testing/playwright.mdx

```diff
@@ -273,4 +273,101 @@ test('ensure that clerk has loaded', async ({ page }) => {
 })
 ```
 
+## Testing authenticated flows
+
+Playwright executes tests in isolated environments called browser contexts. So, each test case runs in a new browser context, which means that the user session is not shared between test cases. However, tests can load existing authenticated state in Playwright.
+
+Following the [Playwright docs on Authentication](https://playwright.dev/docs/auth), you can test authenticated flows without having to sign in for each test case.
+
+The steps below showcase an example of how to save the session storage in a file and load it in your test cases. Of course, you can customize this to fit your needs. You can also have a look at our example repository [here](https://github.com/clerk/clerk-playwright-nextjs).
+
+<Steps>
+  ### Add the storage file in your `.gitingore`
+
+  Create a `playwright/.clerk` directory and add it to your `.gitignore`.
+
+  ```sh {{ filename: 'terminal' }}
+  mkdir -p playwright/.clerk
+  echo $'\nplaywright/.clerk' >> .gitignore
+  ```
+
+  ### Save the session in your setup file
+
+  Save the session in your global setup file. This file will be executed before all projects.
+
+  ```tsx {{ filename: 'global.setup.ts' }}
+  import { clerk, clerkSetup } from '@clerk/testing/playwright'
+  import { test as setup } from '@playwright/test'
+  import path from 'path'
+
+  setup('global setup', async ({}) => {
+    await clerkSetup()
+  })
+
+  const authFile = path.join(__dirname, '../playwright/.clerk/user.json')
+
+  setup('authenticate and save state to storage', async ({ page }) => {
+    // here you can either use a helper function to sign in or do it manually
```

Comment:
just curious, what does it look to "manually" sign in? vs. using a clerk helper

Reply:
"Manually" sign in is how you would sign-in on E2E tests. 
An example could be:
```
await page.goto("/sign-in");
await expect(page.locator("h1")).toContainText("Sign In");
await page.waitForSelector(".cl-signIn-root", { state: "attached" });
await page.locator("input[name=identifier]").fill('USERNAME');
await page.getByRole("button", { name: "Continue", exact: true }).click();
await page.locator("input[name=password]").fill('PASSSWORD');
await page.getByRole("button", { name: "Continue", exact: true }).click();
```